src/components/button-toggle-group/button-toggle-group.component.tsx

```diff
@@ -45,33 +47,46 @@ export interface ButtonToggleGroupProps
   fullWidth?: boolean;
   /** The alignment for the text in the label. */
   labelAlign?: "left" | "right";
-  /** Callback triggered by blur event on the input. */
-  onBlur?: (ev: React.FocusEvent<HTMLInputElement>) => void;
-  /** Callback triggered by change event on the input. */
-  onChange?: (ev: React.ChangeEvent<HTMLInputElement>) => void;
-  /** The value of the Button Toggle Group */
+  /** Callback triggered by blur event on one of the child buttons. */
+  onBlur?: (ev: React.FocusEvent<HTMLButtonElement>) => void;
+  /** Callback triggered by pressing one of the child buttons. Use with controlled components to set the value prop to ev.target.value. */
+  onChange?: (ev: CustomEvent) => void;
```

Comment:
I'd rather avoid introducing more `CustomEvents` to the library, so far they've been a PITA during TS refactors. I think that better approach would be to pass the normal event (React.MouseEvent<HTMLButtonElement> in this case) as a first argument, and then the selected value as a second argument

Reply:
Oh sorry, I've just read your comments/suggested change more carefully and see that you were proposing passing the `value` as a second argument. That's a reasonable idea, although it will mean consumers having to rewrite existing implementations which I'd have preferred to avoid. I'll also have to add a third argument for the name, at least temporarily.